instaclustr/structs.go

```diff
@@ -155,27 +155,24 @@ type Node struct {
 }
 
 type CreateVPCPeeringRequest struct {
-	PeerVpcID     string `json:"peerVpcId"`
-	PeerAccountID string `json:"peerAccountId"`
-	PeerSubnet    string `json:"peerSubnet"`
-	PeerRegion    string `json:"peerRegion,omitempty"`
+	PeerVpcID     string        `json:"peerVpcId"`
```

Comment:
Not going to press this, but could `mapstructure` be used to avoid having to manually map the schema to the request model?

Reply:
Agree with this. And if possible using strongly typed decoding since it is string only.

Reply:
Had some issues with it, it attempted to set both `peerSubnet` and `peerSubnets` even when only one of them was actually defined in the plan, and as a result we got some 400 errors from the API. Spent a bit too much time looking at it, so will be proceeding with the original code that does not use `mapstructure`